presto-iceberg/src/test/java/io/prestosql/plugin/iceberg/TestIcebergMetadataListing.java

```diff
@@ -0,0 +1,197 @@
+    private static final String TEST_SCHEMA = "test_schema";
+    private static final String ICEBERG_TABLE1 = "iceberg_table1";
+    private static final String ICEBERG_TABLE2 = "iceberg_table2";
+    private static final String HIVE_TABLE = "hive_table";
+
+    public TestIcebergMetadataListing()
+    {
+        super(TestIcebergMetadataListing::createQueryRunner);
+    }
+
+    @BeforeClass
+    public void setUp()
+    {
+        @Language("SQL") String createSchema = "CREATE SCHEMA "
+                + HIVE_CATALOG + "." + TEST_SCHEMA;
+
+        @Language("SQL") String createIcebergTable1 = "CREATE TABLE "
+                + ICEBERG_CATALOG + "." + TEST_SCHEMA + "." + ICEBERG_TABLE1
+                + "("
+                + "  _string VARCHAR"
+                + ", _integer INTEGER"
+                + ")";
+
+        @Language("SQL") String createIcebergTable2 = "CREATE TABLE "
+                + ICEBERG_CATALOG + "." + TEST_SCHEMA + "." + ICEBERG_TABLE2
+                + "("
+                + "  _string VARCHAR"
+                + ", _integer INTEGER"
+                + ", _double DOUBLE"
+                + ", _boolean BOOLEAN"
+                + ") "
+                + "WITH ("
+                + "partitioning = ARRAY["
+                + "  '_string'"
+                + ", '_integer'])";
+
+        @Language("SQL") String createHiveTable = "CREATE TABLE "
+                + HIVE_CATALOG + "." + TEST_SCHEMA + "." + HIVE_TABLE
+                + "("
+                + "  _string VARCHAR"
+                + ", _integer INTEGER"
+                + ", _double DOUBLE"
+                + ")";
+
+        assertQuerySucceeds(createSchema);
+        assertQuerySucceeds(createIcebergTable1);
+        assertQuerySucceeds(createIcebergTable2);
+        assertQuerySucceeds(createHiveTable);
+    }
+
+    @AfterClass
+    public void tearDown()
+    {
+        @Language("SQL") String dropSchema = "DROP SCHEMA "
+                + HIVE_CATALOG + "." + TEST_SCHEMA;
+
+        @Language("SQL") String dropIcebergTable1 = "DROP TABLE "
+                + ICEBERG_CATALOG + "." + TEST_SCHEMA + "." + ICEBERG_TABLE1;
+
+        @Language("SQL") String dropIcebergTable2 = "DROP TABLE "
+                + ICEBERG_CATALOG + "." + TEST_SCHEMA + "." + ICEBERG_TABLE2;
+
+        @Language("SQL") String dropHiveTable = "DROP TABLE "
+                + HIVE_CATALOG + "." + TEST_SCHEMA + "." + HIVE_TABLE;
+
+        assertQuerySucceeds(dropIcebergTable1);
+        assertQuerySucceeds(dropIcebergTable2);
+        assertQuerySucceeds(dropHiveTable);
+        assertQuerySucceeds(dropSchema);
+    }
+
+    @Test
+    public void testTableListing()
+    {
+        @Language("SQL") String showIcebergTables = "SHOW TABLES FROM "
+                + ICEBERG_CATALOG + "." + TEST_SCHEMA;
+
+        @Language("SQL") String expected = "VALUES "
+                + "'" + ICEBERG_TABLE1 + "'"
+                + ", '" + ICEBERG_TABLE2 + "'";
```

Comment:
Inline the variable and inline the table names too.

Compare readability of:
```
assertQuery("SHOW TABLES FROM iceberg.test_schema", "VALUES 'iceberg_table1', 'icebert_table2'");
```

with

```
	        @Language("SQL") String showIcebergTables = "SHOW TABLES FROM "
                + ICEBERG_CATALOG + "." + TEST_SCHEMA;

         @Language("SQL") String expected = "VALUES "
                + "'" + ICEBERG_TABLE1 + "'"
                + ", '" + ICEBERG_TABLE2 + "'";

         assertQuery(showIcebergTables, expected);
```

Reply:
Same applies to other tests too.